keep-ui/app/(keep)/workflows/manual-run-workflow-modal.tsx

```diff
@@ -8,7 +8,11 @@ import { useRouter } from "next/navigation";
 import { IncidentDto } from "@/entities/incidents/model";
 import { AlertDto } from "@/entities/alerts/model";
 import { useApi } from "@/shared/lib/hooks/useApi";
-import { showErrorToast } from "@/shared/ui";
+import { Select, showErrorToast } from "@/shared/ui";
+import { Trigger, Workflow } from "@/shared/api/workflows";
+import { components, OptionProps } from "react-select";
```

Comment:
I think we already have Select in ui/components?

Reply:
Yes, we have already used this one in the incidents. In fact, this is a wrapper around react-select.